### DISCUSSION TITLE
SEO Wordpress import experiences

### DISCUSSION THREAD

**Discussion by hubyhuby** [GetPublii/Publii#1473] [Help]
Hi there I am interested in going all publii <3
I want to converte a wordpress website.
My main concern is SEO and URL correctness.
I need all my page and blog url to be the same.

**example Blog Urls :**
/blog/category/design-thinking/
/blog/7-etapes-design-thinking/

**Page url:**

/plateforme-gestion-innovation-collaborative/
/plateforme-gestion-incubateur-projet-innovant/

I am interested in your experience with this migration ?

![image](https://github.com/GetPublii/Publii/assets/26340942/1b19b75d-6101-4bf1-8fa6-60331156a729)

**Comment by sandrobilbeisi** [comment]
For starters, I assume that your main site (canonical) will be in the French language.
To handle the two different sets of URLs (under /blog) and the ones under root you may need two distinct Publii website points.

**Reply by hubyhuby** [reply]
Ho you are right, I forgot my site is in French and english.
Url like
/en
/en/innovation-management-platform-for-large-enterprises/

The english website is smaller there are only pages.

I d better wait for the language pluggin then ...
Really looking forward to this plugin I need language for a new website soon.

If I understand well, there is no way to have the URL /blog imported easily ? 
It would be a real pain to do 2 websites! Just to have the "/blog".
I think there is a static website generator plugin for wordpress...

**Reply by sandrobilbeisi** [reply]
You could have /en/ , /fr/ , /blog/

**Comment by nordseo** [comment]
I'm attempting to build a plugin for Publii > Obsidian Importer. It should work similarly to the WP Importer. Can I fork this component in Publii? As far as I understand, it's part of the Publii binary.
